crates/bevy_asset/src/handle.rs

```diff
@@ -230,6 +231,34 @@ impl<T: Asset> Clone for Handle<T> {
     }
 }
 
+impl<T: Asset + 'static> Lerp for Handle<T> {
+    #[inline(always)]
+    fn lerp_unclamped(a: &Self, b: &Self, t: f32) -> Self {
```

Comment:
I would like to see more descriptive variable names here. I can piece together a = "starting_position", b = "end_position", t = "time_step" from context but that will be harder for beginners and tough to read.

Reply:
from `a` to `b` is kind "standard" in the lerp functions, maybe docs should solve the issue?

alternatively I could agree with `v0` to `v1` meaning value at 0 and 1, what do you think?

Reply:
I'm fine with `a` and `b` if they're standard, so long as they're documented in a way that's attached to the function :)